bazaarvoice/emodb
# delete api-key endpoint postman tests

## What Are We Doing Here?

Adding new postman tests for delete api-key endpoint tests

## How to Test and Verify

1. Run this PR on CI 
2. Verify that new postman tests are executed
3. Verify that all previously executed tests are still running and no new failures have been introduced 
5. Profit

## Risk
Unstable tests are added to test suite

### Level 

`Low`

### Required Testing

`Smoke`
### Risk Summary

Unstable tests can be added to test suite

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
